rskj-core/src/test/java/co/rsk/peg/BridgeSupportReleaseBtcTest.java

```diff
@@ -0,0 +1,463 @@
+
+        when(activationMock.isActive(ConsensusRule.RSKIP146)).thenReturn(true);
+        when(activationMock.isActive(ConsensusRule.RSKIP185)).thenReturn(true);
+
+        bridgeSupport = initBridgeSupport(eventLogger, activationMock);
+
+        Transaction rskTx = buildUpdateTx();
+        bridgeSupport.updateCollections(rskTx);
+
+        verify(repository, never()).transfer(any(), any(), any());
+        verify(eventLogger, never()).logReleaseBtcRequested(any(byte[].class), any(BtcTransaction.class), any(Coin.class));
+        verify(eventLogger, never()).logReleaseBtcRequestReceived(any(), any(), any());
+        verify(eventLogger, never()).logReleaseBtcRequestRejected(any(), any(), any());
+    }
+
+    @Test
+    public void handmade_release_before_rskip_146_185() throws IOException {
+        when(activationMock.isActive(ConsensusRule.RSKIP146)).thenReturn(false);
+        when(activationMock.isActive(ConsensusRule.RSKIP185)).thenReturn(false);
+
+        bridgeSupport.releaseBtc(releaseTx);
+
+        Transaction rskTx = buildUpdateTx();
+        rskTx.sign(new ECKey().getPrivKeyBytes());
+        bridgeSupport.updateCollections(rskTx);
+
+        verify(repository, never()).transfer(any(), any(), any());
+        assertEquals(1, provider.getReleaseTransactionSet().getEntries().size());
+        assertEquals(0, provider.getReleaseRequestQueue().getEntries().size());
+
+        verify(eventLogger, never()).logReleaseBtcRequested(any(byte[].class), any(BtcTransaction.class), any(Coin.class));
+        verify(eventLogger, never()).logReleaseBtcRequestReceived(any(), any(), any());
+        verify(eventLogger, never()).logReleaseBtcRequestRejected(any(), any(), any());
+    }
+
+    @Test
+    public void handmade_release_after_rskip_146() throws IOException {
+        when(activationMock.isActive(ConsensusRule.RSKIP146)).thenReturn(true);
+        when(activationMock.isActive(ConsensusRule.RSKIP185)).thenReturn(false);
+
+        List<LogInfo> logInfo = new ArrayList<>();
+        BridgeEventLoggerImpl eventLogger = new BridgeEventLoggerImpl(bridgeConstants, activationMock, logInfo);
+        bridgeSupport = initBridgeSupport(eventLogger, activationMock);
+
+        bridgeSupport.releaseBtc(releaseTx);
+
+        Transaction rskTx = buildUpdateTx();
+        rskTx.sign(new ECKey().getPrivKeyBytes());
+        bridgeSupport.updateCollections(rskTx);
+
+        verify(repository, never()).transfer(any(), any(), any());
+        assertEquals(1, provider.getReleaseTransactionSet().getEntries().size());
+        assertEquals(0, provider.getReleaseRequestQueue().getEntries().size());
+        ReleaseTransactionSet.Entry entry = (ReleaseTransactionSet.Entry) provider.getReleaseTransactionSet().getEntries().toArray()[0];
+        assertEvent(logInfo, 1, BridgeEvents.RELEASE_REQUESTED.getEvent(), new Object[]{releaseTx.getHash().getBytes(), entry.getTransaction().getHash().getBytes()}, new Object[]{Coin.COIN.value});
```

Comment:
```suggestion
        verify(eventLogger,times(1)).logReleaseBtcRequested(any(byte[].class), any(BtcTransaction.class), any(Coin.class));
```
Since the event content is tested in BridgeEventLoggerImplTest maybe it's enough to simply check the call was made here

Reply:
Done.

Reply:
I would do this on the rest of the tests also

Reply:
Done